### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by JatuFaique**
Please review my alert component.
Screenshot 2022-02-03 at 12-34-23 Alert

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap");
+
+* {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```
Comment (khyatibaria) [inline alert/alert.css]: suggestions color can be kept in common variable.css file and you can use them everywhere else

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  border-radius: 2px;
+  color: white;
+}
+
+.heading {
+  display: inline-flex;
+  align-items: center;
+}
+
+.bs {
```
Comment (khyatibaria) [inline alert/alert.css]: suggestion class name can be more descriptive eg: alert-success

File: alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+body {
+  font-family: "Inter", sans-serif;
+  --fulvous: rgb(217, 126, 28);
+  --fulvous-light: rgba(217, 126, 28, 0.349);
+  --border-danger: rgb(216, 43, 43);
+  --bright-navy-blue: rgba(28, 119, 217, 0.63);
+  --hookers-green: rgb(73, 111, 93);
+  --border-succes: rgb(73, 216, 149);
+  --mint-cream: hsla(120, 100%, 98%, 1);
```
Comment (khyatibaria) [inline alert/alert.css]: suggestion try following one method for color value if using rgba, use it everywhere

File: alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    </div>
+   </section>
+   <h2>Alerts</h2>
+   <section class="alerts-section disp-flex">
+    <div class="al in disp-flex-col">
+        <div class="heading"> 
+            <i class="fas fa-info-circle"></i>
+            <h3>Info</h3> 
+        </div> 
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ex expedita delectus distinctio aut dolor eveniet </p>
```
Comment (khyatibaria) [inline alert/alert.html]: try not to change/add html for closeable alert handle it with css

**Review by khyatibaria — CHANGES_REQUESTED** [review]

File: alert/alert.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+* {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  --fulvous: rgb(217, 126, 28);
```
Comment (zabihshaik) [inline alert/alert.css]: Same suggestion as Khyati to use Variables

File: alert/alert.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  --fulvous-light: rgba(217, 126, 28, 0.349);
+  --border-danger: rgb(216, 43, 43);
+  --bright-navy-blue: rgba(28, 119, 217, 0.63);
+  --hookers-green: rgb(73, 111, 93);
+  --border-succes: rgb(73, 216, 149);
+  --mint-cream: hsla(120, 100%, 98%, 1);
+  --black-chocolate: hsla(57, 100%, 4%, 1);
```
Comment (zabihshaik) [inline alert/alert.css]: Better to follow a consistent method. Using either HEX values, or rgb everywhere.

File: alert/alert.css [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  display: inline-flex;
+  align-items: center;
+}
+
+.bs {
+  background-color: var(--fulvous-light);
+  border: 1px solid var(--fulvous);
+}
+
+.ss {
```
Comment (zabihshaik) [inline alert/alert.css]: Better to use understandable class names

File: alert/alert.html [hunk 8]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/51b1b3d424.js" crossorigin="anonymous"></script>
```
Comment (zabihshaik) [inline alert/alert.html]: Better to use meta tag with your name in content

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a few suggestions in comments. Please check.

**Pull request merged**